releasewarrior/cli.py

```diff
@@ -189,6 +191,27 @@ def graphid(graphid, product, version, phase, logger=LOGGER, config=CONFIG):
     write_and_commit(data, data_path, wiki_path, commit_msg, logger, config)
 
 
+@cli.command()
+@click.argument('revision')
+@click.argument('product', type=click.Choice(['firefox', 'devedition', 'fennec', 'thunderbird']))
+@click.argument('version')
+def revision(revision, product, version, logger=LOGGER, config=CONFIG):
```

Comment:
I wonder, since this is known when we run `newbuild` if this should just be an additional arg there rather than running a separate cmd like this.

This seems like something we would want rw to know every time.

Reply:
Agreed!